cv32/sim/uvmt_cv32/Makefile

```diff
@@ -153,8 +211,6 @@ dsim-no-firmware: dsim-comp
 		+UVM_TESTNAME=$(UVM_TESTNAME)
 #		+verbose
 
-dsim-sanity: dsim-hello_world
```

Comment:
The sanity rule was deliberately added.  Its redundant now (just hello_world), but will be added to over time.  I would like to keep it.